oem/ibm/configurations/pdr/ibm,rainier-1s4u/11.json

```diff
@@ -614,6 +614,44 @@
                         }
                     ]
                 },
+                {
```

Comment:
why do you have these changes in the PR?

Reply:
corrected.